### PR TITLE
[Issue #1112] Redirect to pregen bundles only ASR clients.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@pmac will merge to try out pregen bundles but i'd like your retrospective review when you have the time. thanks!

File: snippets/base/views.py [hunk 1]
```diff
@@ -65,6 +66,10 @@ def fetch_snippet_pregen_bundle(request, **kwargs):
         f'{settings.MEDIA_BUNDLES_PREGEN_ROOT}/{product}/{channel}/'
         f'{locale}/{distribution}.json'
     )
+
+    if not default_storage.exists(filename):
```
Comment (glogiotatidis) [inline snippets/base/views.py]: A test run indicates that this needs to be cached, will investigate further

File: snippets/base/views.py [hunk 2]
```diff
@@ -47,13 +47,14 @@ def template_name(self):
 
 
 def fetch_snippets(request, **kwargs):
-    if settings.USE_PREGEN_BUNDLES:
+    if settings.USE_PREGEN_BUNDLES and kwargs['startpage_version'] == 6:
```
Comment (pmac) [inline snippets/base/views.py]: Only version 6 or `>= 6`?

Comment (glogiotatidis) [reply snippets/base/views.py]: Will keep '6' for now. Who knows what `7` will look like

Comment (pmac) [reply snippets/base/views.py]: Fair.

**Pull request merged**